includes/forms/class-llms-forms.php

```diff
@@ -64,6 +65,7 @@ private function __construct() {
 		add_filter( 'render_block', array( $this, 'render_field_block' ), 10, 2 );
 		add_filter( 'llms_get_form_post', array( $this, 'maybe_load_preview' ) );
 
+		add_filter( 'llms_get_form_blocks', array( $this, 'maybe_add_required_email_field_block' ), 999, 3 );
```

Comment:
I think this hook & method should be moved to the `LLMS_Forms_Dynamic_Fields` class

Reply:
not as it is right now, because I need $this->is_block_visible()

Reply:
mmmm okay... well i think to keep the "dynamic" stuff in a single class makes sense... Make the method public then!

Reply:
ok